cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -99,6 +99,7 @@ func TestAddFlags(t *testing.T) {
 		"--cloud-provider=azure",
 		"--cors-allowed-origins=10.10.10.100,10.10.10.200",
 		"--contention-profiling=true",
+		"--disable-http2",
```

Comment:
it's not clear whether this flag controls http2 serving or http2 in the various clients the API server creates/uses ... can we disambiguate this?